Source/Core/Common/GL/GLInterface/BGL.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <Window.h>
+#include <GLView.h>
+
+#include "Common/GL/GLInterfaceBase.h"
+
+class cInterfaceBGL : public cInterfaceBase
+{
+private:
```

Comment:
I copied this code mostly from the AGLInterface. Might want to clean that up then.